events/events.go

```diff
@@ -0,0 +1,15 @@
+package events
+
+import "fmt"
+
+var DEBUG = true
+
+func Info(key string, message string) {
+	fmt.Println(key + " : " + message)
```

Comment:
How about using something like logrus (https://github.com/sirupsen/logrus) for logging rather than prints if the plan is to expand this package?

Reply:
Yep, happy to include this. I'd sort have thought we might use an abstraction like this to add in event monitoring, sending event to some sort of external tool. For now though, logging will be fine.

Reply:
Ooh, excellent. Loving the integration with the other systems. :+1: